blog/urls.py

```diff
@@ -0,0 +1,22 @@
+from django.urls import path, include
+
+from blog.views import (AuthorCreateView,
+                        PostDetailView,
+                        CreateCommentView,
+                        PostListView,
+                        IndexListView
+                        )
```

Comment:
Place each import on a new line so that there would be no imports on the same line with opening and closing parenthesis. It will help to improve the code style and readability